crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -21,6 +21,7 @@ module sui::sui_system {
     use sui::epoch_time_lock::EpochTimeLock;
     use sui::epoch_time_lock;
     use sui::pay;
+    use sui::event;
```

Comment:
noob q, how / when are functions in `sui_system` called, by a blessed validator or sth else?

Reply:
Depends on which function you are asking about. Delegation related ones are callable by anybody. Validator management/self-staking ones are callable by validators. `advance_epoch` is special and only called during reconfig from Rust.